### PR TITLE
Fix defect #1989

### LINKED ISSUE #1989: Button from ibm-tab-headers missing aria-label attribute
## Detailed description

> Describe in detail the issue you're having.

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug
> Is this issue related to a specific component?
tabs

> What browser are you working in?
Firefox 92.0.1 (64-bit)

> What version of the Carbon Design System are you using?
    "carbon-components": "10.31.0",
    "carbon-components-angular": "4.46.0",

## Steps to reproduce the issue

1. zoom in the panel that contains ibm-tab-header component 
2. Run Accessibility Assessment

## Additional information

* Screenshots or code
button has no label
aria - label

* Notes

### PR DESCRIPTION
**Pull request by GSSCDylan**
Closes IBM/carbon-components-angular#

{{short description}}

#### Changelog

**Changed**
tabs
* {{change thing}}
Add title attribute for button of tab-headers, so Jaws can tell the user what's the button used for

### CODE REVIEW

File: src/tabs/tab-headers.component.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ import { Tab } from "./tab.component";
 			<button
 				#leftOverflowNavButton
 				type="button"
+				title="previous"
```
[the remark this answers is not in the document]
Comment (GSSCDylan) [reply src/tabs/tab-headers.component.ts]: Fix it as your suggestion, thank you! Please help to review it again!

File: src/ui-shell/header/header-item.component.ts [hunk 2]
```diff
@@ -14,13 +14,13 @@ import { Router } from "@angular/router";
 @Component({
 	selector: "ibm-header-item",
 	template: `
-		<li style="height: 100%">
+		<li style="height: 100%"
+			role="menuitem">
```
Comment (Donisius) [inline src/ui-shell/header/header-item.component.ts:18]:
```suggestion
		<li
			style="height: 100%"
			role="menuitem">
```

**Review by Donisius — CHANGES_REQUESTED** [review]
Thanks for doing this! Just have a couple of small changes!

**Review by Donisius — APPROVED** [review]

**Comment by zvonimirfras** [comment]
@GSSCDylan good to go once build issues are resolved 👍

**Comment by GSSCDylan** [comment]
> @GSSCDylan good to go once build issues are resolved 👍

Thanks, I'm not familiar with build process, is there anything need I to take action?

**Comment by zvonimirfras** [comment]
> > @GSSCDylan good to go once build issues are resolved 👍
> 
> Thanks, I'm not familiar with build process, is there anything need I to take action?

Luckily, you don't need to be familiar with the process to read the [build log](https://app.travis-ci.com/github/IBM/carbon-components-angular/builds/248901929) 😉 

From there, it says `NullInjectorError: No provider for I18n` which sounds like there's no provider of `I18n`.
Looking at your PR, you _did_ introduce the `I18n` to tabs, and tabs module (`src/tabs/tabs.module.ts`) doesn't seem to already have the provider in.

This looks like a standard Angular workflow so it would be a good first step to look at.

You can generate the build log locally by running `npm run build`. It will give you insights what might be going wrong in case the build fails.

**Comment by GSSCDylan** [comment]
> > > @GSSCDylan good to go once build issues are resolved 👍
> > 
> > 
> > Thanks, I'm not familiar with build process, is there anything need I to take action?
> 
> Luckily, you don't need to be familiar with the process to read the [build log](https://app.travis-ci.com/github/IBM/carbon-components-angular/builds/248901929) 😉
> 
> From there, it says `NullInjectorError: No provider for I18n` which sounds like there's no provider of `I18n`. Looking at your PR, you _did_ introduce the `I18n` to tabs, and tabs module (`src/tabs/tabs.module.ts`) doesn't seem to already have the provider in.
> 
> This looks like a standard Angular workflow so it would be a good first step to look at.
> 
> You can generate the build log locally by running `npm run build`. It will give you insights what might be going wrong in case the build fails.

fix the issue by import I18nModule into tabs.component.spec and tab.headers.component.spec

**Comment by GSSCDylan** [comment]
> > > @GSSCDylan good to go once build issues are resolved 👍
> > 
> > 
> > Thanks, I'm not familiar with build process, is there anything need I to take action?
> 
> Luckily, you don't need to be familiar with the process to read the [build log](https://app.travis-ci.com/github/IBM/carbon-components-angular/builds/248901929) 😉
> 
> From there, it says `NullInjectorError: No provider for I18n` which sounds like there's no provider of `I18n`. Looking at your PR, you _did_ introduce the `I18n` to tabs, and tabs module (`src/tabs/tabs.module.ts`) doesn't seem to already have the provider in.
> 
> This looks like a standard Angular workflow so it would be a good first step to look at.
> 
> You can generate the build log locally by running `npm run build`. It will give you insights what might be going wrong in case the build fails.

fix the issue by import I18nModule into tabs.component.spec and tab.headers.component.spec

**Review by zvonimirfras — APPROVED** [review]

**Pull request merged**